Detectors/TPC/reconstruction/include/TPCReconstruction/KrBoxClusterFinder.h

```diff
@@ -163,6 +170,17 @@ class KrBoxClusterFinder
     mMaxClusterSizeTime = maxClusterSizeTime;
   }
 
+  void fillADCValue(int cru, int rowInSector, int padInRow, int timeBin, float adcValue);
+  void resetADCMap();
+
+  void createInitialMap(std::vector<o2::tpc::Digit>& eventSector);
+  void addTimeSliceToMap();
+  void popFirstTimeSliceFromMap();
+  void fillADCValueInTempSlice(int cru, int rowInSector, int padInRow, float adcValue);
+  void fillTempTimeSlice(std::vector<o2::tpc::Digit>& eventSector, const int timeSlice);
+  void clearTempTimeSlice();
+  const std::vector<KrCluster>& loopOverSector(std::vector<o2::tpc::Digit>& eventSector);
```

Comment:
```suggestion
  void loopOverSector(std::vector<o2::tpc::Digit>& eventSector);
```
I think it doesn't make sense to return mClusters, since there is already a getter for this.